collector/os.go

```diff
@@ -131,6 +140,16 @@ func (c *OSCollector) collect(ch chan<- prometheus.Metric) (*prometheus.Desc, er
 		float64(dst[0].FreePhysicalMemory*1024), // KiB -> bytes
 	)
 
+	time := dst[0].LocalDateTime
+	timeZoneName, _ := time.Zone()
+	
+	ch <- prometheus.MustNewConstMetric(
+		c.LocalDateTime,
+		prometheus.GaugeValue,
+		float64(time.Unix()),
```

Comment:
Local time implies that timezones are involved, which is not the case for unixtime. This should be just called "time".

The timezone should go on a separate 1-valued metric.